### ISSUE TITLE
converting input image to grayscale

### ISSUE THREAD

**Issue by KushGrandhi** [KushGrandhi/fashion-MNIST#1]

**Comment by Palaash05** [comment]
I would be glad to work on this issue. I request you to please assign it to me.

**Comment by KushGrandhi** [comment]
@Palaash05 go ahead

**Comment by Palaash05** [comment]
@KushGrandhi  could you please add hacktoberfest label to your repo

**Comment by Palaash05** [comment]
@KushGrandhi  I have submitted a PR #2 please check it and add the hacktober fest label

**Comment by KushGrandhi** [comment]
check if the label is added in a correct way

**Comment by Palaash05** [comment]
I don't think so, because its still not showing accepted. I guess you should add the label to your repo.

**Comment by Palaash05** [comment]
You can also add the hacktoberfest accepted label to the PR

**Comment by Palaash05** [comment]
@KushGrandhi  could you please reply. Please add the hacktoberfest label to the repo or hacktoberfest accepted label to my PR.
